config.json

```diff
@@ -625,6 +625,14 @@
         ],
         "difficulty": 3
       },
+      {
+        "slug": "square-root",
+        "name": "Square Root",
+        "uuid": "8960514f-17af-4009-b334-d6bfc4722c80",
+        "practices": [],
```

Comment:
This would be a perfect fit for #692 but in the mean time, we can leave it empty